lib/dal-materialized-views/src/component.rs

```diff
@@ -77,6 +80,14 @@ pub async fn assemble(ctx: DalContext, component_id: ComponentId) -> crate::Resu
     let resource_value_attribute_value_id =
         Component::attribute_value_for_prop(ctx, component_id, &["root", "resource_value"]).await?;
 
+    let dal_component_diff = Component::get_diff(ctx, component_id).await?;
+    let mut diff = None;
+    if let Some(diff_code) = dal_component_diff.diff {
+        diff = diff_code.code;
+    }
```

Comment:
Only because you asked 😉

```suggestion
    let diff = dal_component_diff.diff.map(|diff_code| diff_code.diff);
```